xml/containers-buildah-overview.xml

```diff
@@ -0,0 +1,209 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+    [
+<!ENTITY % entities SYSTEM "entity-decl.ent">
+%entities;
+]>
+<chapter xml:id="cha-buildah-overview" xml:lang="en"
+    xmlns="http://docbook.org/ns/docbook" version="5.0"
+    xmlns:xi="http://www.w3.org/2001/XInclude"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>&buildah; Overview</title>
+ <para>
+  <link xlink:href="https://buildah.io/">&buildah;</link> is tool for building
+  OCI-compliant container images. &buildah; can handle the following tasks.
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    Create containers from scratch, or from an existing image.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Create an image from a working container or via Dockerfile.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Build images in the OCI image format or the traditional upstream &deng;
+    image format.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Mount a working container's root filesystem for manipulation.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Use the updated contents of a container's root filesystem as a filesystem
+    layer to create a new image.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Delete a working container or an image and rename a local container.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <para>
+  Compared to &deng;, &buildah; has several important advantages.
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    &buildah; adheres the UNIX philosophy by focusing only on one task:
+    building container images.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The tool makes it possible to mount a working container's filesystem, so it
+    becomes accessible by host.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The process of building container images using &buildah; can be automated
+    using scripts.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Similar to &podman;, &buildah; supports the rootless mode.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    It is possible to build images inside a container without mounting the
+    &docker; socket, which improves security.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <sect1 xml:id="cha-podman-buildah">
+  <title>&podman; and &buildah;</title>
+
+  <para>
+   &podman; and &buildah; are complimentary tools for working with OCI images
+   and containers. While &buildah; focuses on building images, &podman; offers
+   commands and functions for maintaining and modifying images. It also makes
+   it possible to create, run, and maintain containers created from images.
+   Although &podman; allows you to build images via Dockerfile, it uses
+   &buildah;'s golang API to accomplish that.
```

Comment:
This reads like buildah needs to be installed to use podman, but AFAIK that's not true.

Reply:
I agree, I think the `Although... accomplish that` line can be confusing for a new user. I would just point out that podman build images using `Dockerfile` like the docker container engine does.
The end user doesn't care about the implementation details.

Reply:
Indeed. `podman build` users an embedded/vendored copy of buildah to give `podman`  the equivalent functionality of `docker build`, but you do not need buildah to be installed.

If I were to describe it simply I would say that `podman` providers a `docker build`-like experience for building containers, whereas `buildah` provides an expanded range of options and capabilities for building containers. Whether I'd document the fact `podman` uses the `buildah` codebase to accomplish this seems like a cute optional implimentation detail to me.